xlators/protocol/auth/addr/src/addr.c

```diff
@@ -45,6 +45,16 @@ compare_addr_and_update(char *option_str, char *peer_addr, char *subvol,
     char match = 0;
     int length = 0;
     int ret = 0;
+    char *hostname = NULL;
+
+    ret = gf_get_hostname_from_ip(peer_addr, &hostname);
+    if (ret) {
+        gf_log(subvol, GF_LOG_ERROR, "DNS ip to host %s resolution is failed",
+               peer_addr);
+        GF_FREE(hostname);
```

Comment:
Is this needed, if we clean it up in 'out' ?

Reply:
Not required